ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -413,15 +413,7 @@ public struct Avatar: View, TokenizedControlView {
         }
 
         static func initialsHashCode(fromPrimaryText primaryText: String?, secondaryText: String?) -> Int {
-            var combined: String
-            if let secondaryText = secondaryText, let primaryText = primaryText, secondaryText.count > 0 {
-                combined = primaryText + secondaryText
-            } else if let primaryText = primaryText {
-                combined = primaryText
-            } else {
-                combined = ""
-            }
-
+            let combined = (primaryText ?? "") + (secondaryText ?? "")
```

Comment:
Is concatenating two empty strings different than just one empty string in Swift? Just double checking to make sure this is semantically equivalent to what it's replacing!

Reply:
Good question! Yep, output is identical.